src/main/java/Duke/Tasks/DeadlineTask.java

```diff
@@ -0,0 +1,26 @@
+package Duke.Tasks;
+
+import Duke.Parser;
+
+public class DeadlineTask extends Task {
+
+    private String by;
+
+    String str;
+    String deadline;
+    public DeadlineTask(String task) {
+        super(task);
+        String[] strArr = task.split("/");
+        str = strArr[0];
+        deadline = strArr[1].split(" ", 2)[1];
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[D] | %s | %s | %s",
+                this.isDone() ? "[X]" : "[ ]",
+                str, Parser.dateToString(deadline));
```

Comment:
Will it be better if the comma follows the next line in a line break. i.e. ```, this.isDone() ? "[X]" : "[ ]"``` for the second line?